packages/ui/src/components/va-image/hooks/useNativeImgAttributes.ts

```diff
@@ -0,0 +1,47 @@
+import { computed, type ExtractPropTypes, type PropType } from 'vue'
+import pick from 'lodash/pick.js'
+
+const REFERRER_POLICY_OPTIONS = ['no-referrer', 'no-referrer-when-downgrade', 'origin', 'origin-when-cross-origin', 'same-origin', 'strict-origin', 'strict-origin-when-cross-origin', 'unsafe-url'] as const
+const LOADING_OPTIONS = ['lazy', 'eager'] as const
+const CROSSORIGIN_OPTIONS = ['anonymous', 'use-credentials'] as const
+const DECODING_OPTIONS = ['auto', 'sync', 'async'] as const
+const FETCH_PRIORITY_OPTIONS = ['auto', 'high', 'low'] as const
+
+export const validateImgProp = (v: string, arr: Readonly<string[]>) => arr.includes(v)
+
+export const useNativeImgAttributesProps = {
+  src: { type: String, required: true },
+  alt: { type: String, default: '' },
+  title: { type: String, default: '' },
+  sizes: { type: String, default: '' },
+  srcset: { type: String, default: '' },
+  notDraggable: { type: Boolean, default: false },
```

Comment:
Is this native img attribute?

Reply:
`draggable` is global and also suitable for `img` imo.

Reply:
Then it must be draggable, not notdraggable

Reply:
It's draggable by default (specification). We discussed and decided to name it `not-draggable` in purpose not to use it everywhere like `:draggable="false"` but just `not-draggable`.